src/app/workflow/workflow.component.ts

```diff
@@ -312,18 +311,23 @@ export class WorkflowComponent extends Entry implements AfterViewInit, OnInit {
     this.selectTab(this.validTabs.indexOf('versions'));
   }
 
+  public getWorkflowSubclass(entryType: EntryType): WorkflowSubClass {
+    let subclass: WorkflowSubClass;
```

Comment:
If you make this a switch statement, and then if we're ever able to turn on StrictNullCheck we can have the compiler catch any missing values: https://stackoverflow.com/a/55532967/257235